website/blog/2024-08-14-otel-context-propagation-in-ferretdb.md

```diff
@@ -0,0 +1,204 @@
+---
+slug: otel-context-propagation-in-ferretdb
+title: 'OpenTelemetry context propagation in FerretDB'
+authors: [elena]
+description: >
+  In this blog post, we demonstrate how to pass tracing context to queries in FerretDB using OpenTelemetry.
+tags: [observability]
+---
+
+In today's world of distributed systems, achieving reliability depends on various factors, one of which is effective observability.
+OpenTelemetry (OTel) has emerged as a standard for distributed tracing, but passing tracing data to databases remains a significant challenge,
+particularly with document databases like MongoDB.
+At FerretDB, we're committed to addressing this challenge.
+
+<!--truncate-->
```

Comment:
Let's have the "truncate" after the first sentence.

Reply:
Shouldn't we give the lead paragraph before `truncate`? I think, this is the lead paragraph :)

Reply:
Not the lead paragraph, just like a sentence or 2 is okay.
<img width="1336" alt="Screenshot 2024-08-13 at 10 02 41" src="https://github.com/user-attachments/assets/e5ff4b94-d084-40d4-b78f-7a6a38e67caf">
uce the

Reply:
The "truncate" should be after the first sentence so it's not too wordy on the homepage. Don't worry about the intro; the metadata description is picked up for search engines. This one is just for the FerretDB blog.